packages/pn-personagiuridica-webapp/src/components/Deleghe/DelegationsElements.tsx

```diff
@@ -261,13 +261,15 @@ export const Menu: React.FC<Props> = ({ menuType, id, userLogged, row, onAction
       )}
 
       <IconButton
+        id="delegation-menu-icon"
```

Comment:
Check that these ids are unique. The component is used multiple times into Delegation page

Reply:
insert id passed with props of component